apps/total-typescript/src/pages/invoices/[merchantChargeId].tsx

```diff
@@ -190,31 +220,45 @@ const Invoice: React.FC<
                 </tr>
               </thead>
               <tbody>
-                {/* {map(sitePurchases, (purchase) => (
-                  <InvoiceItem key={purchase.guid} purchase={purchase} />
-                ))} */}
-                <tr className="table-row">
-                  <td>
-                    {process.env.NEXT_PUBLIC_SITE_TITLE} ({product.name})
-                    {/* {date} */}
-                  </td>
-                  <td>
-                    {charge.currency.toUpperCase()} {amount}
-                  </td>
-                  <td>1</td>
-                  <td className="text-right">
-                    {amount === null
-                      ? `${charge.currency.toUpperCase()} 0.00`
-                      : `${charge.currency.toUpperCase()} ${amount}`}
-                  </td>
-                </tr>
+                {bulkCoupon ? (
+                  <tr className="table-row">
+                    <td>{product.name}</td>
+                    <td>
+                      {charge.currency.toUpperCase()}{' '}
+                      {formatUsd(charge.amount / 100 / bulkCoupon.maxUses)}
```

Comment:
couldn't think of better way of getting a price per seat 🤔

Reply:
This will work for the first bulk purchase, but if they add more seats on to an existing team, the numbers will get weird since `maxUses` will equal `originalPurchaseQuantity + newPurchaseQuantity`.

You could get the exact quantity for this purchase from Stripe. The commerce server does this.

https://github.com/skillrecordings/products/blob/3b813606e4ca720e422139f53e381e5eca7911d6/packages/commerce-server/src/record-new-purchase.ts#L46

<img width="618" alt="CleanShot 2022-11-30 at 09 38 30@2x" src="https://user-images.githubusercontent.com/694063/204842277-b13e86be-52ea-4230-9ec1-01d13f2dcfea.png">